tests/Filesystem/ClassFinderTest.php

```diff
@@ -24,9 +24,10 @@ public function testFindInterfaces()
     public function testFindClasses()
     {
         $classes = $this->finder->findClasses(__DIR__ . '/fixtures/finder');
+        sort($classes);
```

Comment:
@loonytoons is common to just use `sort` without a flag? e.g. `SORT_REGULAR` (the default).

Reply:
yeah, it just uses the default `SORT_REGULAR`